datatableview/datatables.py

```diff
@@ -326,8 +326,11 @@ def normalize_config_ordering(self, config, query_config):
 
             ordering.append('%s%s' % (sort_modifier, column.name))
 
-        if not ordering and config['model']:
-            return config['model']._meta.ordering
+        if not ordering:
```

Comment:
if ordering not in query string, use Datatable class', and if that is not valid either, use model's.